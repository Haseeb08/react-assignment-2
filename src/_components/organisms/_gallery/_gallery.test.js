import React from "react";
import { create } from "react-test-renderer";
import {
  render,
  fireEvent,
} from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Gallery from "./_gallery";

const image = [
    {
    id: "1",
    name: "React",
    url:
      "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRWXoLhb1QGq1B52D9rNOdWzmIdy0IVkmKk9Q&usqp=CAU",
  },
  {
    id: "2",
    name: "React",
    url:
      "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRWXoLhb1QGq1B52D9rNOdWzmIdy0IVkmKk9Q&usqp=CAU",
  }
];
describe("renders gallery ", () => {
  test("Matches the snapshot", () => {
    const input = create(<Gallery allImages={image} />);
    expect(input.toJSON()).toMatchSnapshot();
  });
});