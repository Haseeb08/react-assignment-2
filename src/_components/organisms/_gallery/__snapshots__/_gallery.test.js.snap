// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders gallery  Matches the snapshot 1`] = `
<div
  className="makeStyles-main-1"
>
  <div
    className="makeStyles-allImages-2"
  >
    <div
      className="makeStyles-box-3"
    >
      <div
        className="makeStyles-image-4"
        style={
          Object {
            "backgroundImage": "URL(https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRWXoLhb1QGq1B52D9rNOdWzmIdy0IVkmKk9Q&usqp=CAU)",
          }
        }
      />
      <div
        className="makeStyles-text-5"
      >
        <h6
          className="MuiTypography-root MuiTypography-subtitle1"
        >
          React
        </h6>
      </div>
    </div>
    <div
      className="makeStyles-box-3"
    >
      <div
        className="makeStyles-image-4"
        style={
          Object {
            "backgroundImage": "URL(https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRWXoLhb1QGq1B52D9rNOdWzmIdy0IVkmKk9Q&usqp=CAU)",
          }
        }
      />
      <div
        className="makeStyles-text-5"
      >
        <h6
          className="MuiTypography-root MuiTypography-subtitle1"
        >
          React
        </h6>
      </div>
    </div>
  </div>
</div>
`;
